prow/cmd/peribolos/main_test.go

```diff
@@ -1264,26 +1264,30 @@ func TestConfigureTeamMembers(t *testing.T) {
 		remove         sets.String
 		addMembers     sets.String
 		addMaintainers sets.String
+		ignoreInvitees bool
 		invitees       sets.String
 		team           org.Team
 		slug           string
 	}{
 		{
-			name: "fail when listing fails",
-			slug: "some-slug",
-			err:  true,
+			name:           "fail when listing fails",
+			slug:           "some-slug",
+			ignoreInvitees: false,
```

Comment:
nit: in prow we usually leave out the explicit zero values from initializers

Reply:
👍 updated